@cbrochtrup Add FIRE_PAGER environment variable for PAGER functionality

cbrochtrup opened the pull request:
Implement [this](https://github.com/google/python-fire/issues/188#issuecomment-820626046) comment.

Fixes #188

cbrochtrup commented:
@googlebot I signed it!

claui commented on fire/console/console_io.py:
```diff
@@ -83,7 +83,11 @@ def More(contents, out, prompt=None, check_pager=True):
+    if fire_pager:
+      # FIRE_PAGER takes precedence to PAGER
+      pager = fire_pager
+    elif pager == '-':
```
How about allowing `FIRE_PAGER=-`, too, so it works consistently like `PAGER`?

```suggestion
    if pager == '-':
```

cbrochtrup replied:
Ope, good catch, we definitely want this. I didn't comprehend this properly. Thanks!

cbrochtrup commented:
> We found a Contributor License Agreement for you (the sender of this pull request), but were unable to find agreements for all the commit author(s) or Co-authors. If you authored these, maybe you used a different email address in the git commits than was used to sign the CLA ([login here](https://cla.developers.google.com/) to double check)? If these were authored by someone else, then they will need to sign a CLA as well, and confirm that they're okay with these being contributed to Google.
> In order to pass this check, please resolve this problem and then comment `@googlebot I fixed it.`. If the bot doesn't comment, it means it doesn't think anything has changed.
> 
> information_source **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fgoogle%2Fpython-fire%2Fpull%2F337) for more info**.

@claui, since I added your suggestion can you sign the CLA please :smile:

claui commented:
@googlebot I consent.

claui commented:
@cbrochtrup I tried everything but it won’t let me. Google doesn’t like the email address GitHub has assigned to my suggestion.

Feel free to revert my suggestion and re-apply it yourself.

cbrochtrup commented:
> @cbrochtrup I tried everything but it won’t let me. Google doesn’t like the email address GitHub has assigned to my suggestion.
> 
> Feel free to revert my suggestion and re-apply it yourself.

Ah shoot, I'm sorry Google isn't accepting your cla. Thanks for your suggestion.
